test/rulesets/LinearAlgebra/structured.jl

```diff
@@ -112,25 +96,23 @@
         m = 3
         @testset "$f(::Matrix{$T})" begin
             A = randn(T, n, m)
-            Ā = randn(T, n, m)
             Y = f(A)
             Ȳ_mat = randn(T, m, n)
             Ȳ_composite = Composite{typeof(Y)}(parent=collect(f(Ȳ_mat)))
 
-            rrule_test(f, Ȳ_mat, (A, Ā))
+            test_rrule(f, A; output_tangent=Ȳ_mat)
 
             _, pb = rrule(f, A)
             @test pb(Ȳ_mat) == pb(Ȳ_composite)
```

Comment:
I wonder if i we can delete this and just do
```suggestion
test_rrule(f, A)
```
 since a composite is the default

Reply:
`test_rrule(f, A)` doesn't work, I've just spent an hour trying to figure out why but giving up now. Should I write an issue in ChainRulesTestUtils and move on with this?

Reply:
yes